### DISCUSSION TITLE
Confidence interval for sceptre LFC estimates?

### DISCUSSION THREAD

**Discussion by ejagoda14** [Katsevich-Lab/sceptre#160] [Feature requests]
Hi,

I was wondering if there is any way to produce a confidence interval or some other value that can be used for error bars on the sceptre effect size calculation? I think I asked about this a while back, but wondering if any more thought has been put in on this front? It would be very helpful to have this for comparison with other effect size estimates. 

Thanks!
Evvie

**Comment by ekatsevi** [comment]
We agree that this feature is needed. We have put it in our queue and will do our best to address it.

**Reply by ejagoda14** [reply]
thank you!

**Comment by ekatsevi** [comment]
This is now available in sceptre. Since version 0.10.3, association analysis results include `se_fold_change`, the standard error of the fold-change estimate, alongside `fold_change`.
